WordPress/Classes/Utility/Blogging Reminders/BloggingRemindersScheduler.swift

```diff
@@ -19,6 +19,8 @@ extension InteractiveNotificationsManager: PushNotificationAuthorizer {
 ///
 class BloggingRemindersScheduler {
 
+    static let notificationCategoryIdentifier = "blogging-reminder-weekly"
```

Comment:
I don't love that this is duplicated here and in InteractiveNotificationsManager, but I couldn't easily see a nicer way of doing it. I think we'd have to make all the notification categories there public, which might be okay?

Reply:
Yes, IMHO they should be public, since notification requests can be initiated outside of `InteractiveNotificationsManager`.